contracts/CO30_Munson_Strange_Bed_Fellows_Part2.chernarus/initPlayerLocal.sqf

```diff
@@ -0,0 +1,41 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+[
+    // Add additional files here
+    // if you create "edit_me/briefing/example.html, then add "Example" here
+] call mission_fnc_briefing;
+
+[
+    laptop,
+    1,  // texture source
+    100, // size in GB
+    300,  // time to upload in seconds
+    {
+        systemChat "Server: Upload Started...";
```

Comment:
No `systemChat`, that is just part of the example code, These also won't show up in multiplayer if you did leave them in